docs/architecture/dapr-for-net-developers/secrets-management.md

```diff
@@ -730,7 +725,7 @@ The Dapr secrets management building block provides capabilities for storing and
 
 The building block supports several different secret stores and hides their complexity with the Dapr secrets API.
 
-The Dapr .NET SDK provides a `DaprClient` object to retrieve secrets. It also includes a .NET configuration provider that adds secrets to the .NET Core configuration system. Once loaded, you can consume these secrets in your .NET code.
+The Dapr .NET SDK provides a `DaprClient` object to retrieve secrets. It also includes a .NET configuration provider that adds secrets to the .NET configuration system. Once loaded, you can consume these secrets in your .NET code.
 
 You can use secret scopes to control access to specific secrets.
 
```

Comment:
There's a closing parenthesis missing at the end of line 738 that causes the Next button not to render in the preview: https://review.docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/secrets-management?branch=pr-en-us-27146#references

Reply:
I will fix this.